core/controllers/cron_test.py

```diff
@@ -338,6 +338,8 @@ def test_run_cron_to_hard_delete_models_marked_as_deleted(self):
             id=admin_user_id,
             exploration_ids=[],
             collection_ids=[],
+            story_ids=[],
+            learnt_topic_ids=[],
```

Comment:
Just to check, weren't there any cron jobs for IncompleteActivities?

Reply:
No, there weren't any.